### PR TITLE
Ft add acados planner

### PR DESCRIPTION
**Pull request by c-salmi**
Adds a simple point mass acados planner.

To test:
- Install acados_template as instructed [here](https://github.com/acados/acados/tree/master/interfaces/acados_template)
- Go to `cd experiments/1_fabric_mpc/pointMass`
- Run `runner -c setup/exp.yaml -p setup/acados_mpc.yaml --render`

### CODE REVIEW

**Comment by maxspahn** [comment]
Great work. The installation was very smooth. 
You can even install it by adding it to the `pyproject.toml` using `poetry add <path-to-acados-template>.

However, I realized that randomized goals cannot be respected. 

`runner -c setup/exp.yaml -p setup/acados_mpc.yaml --random-obst` 

still uses the not-randomized obstacles.

I will further investigate that.

**Review by maxspahn — CHANGES_REQUESTED** [review]
Amazing, @c-salmi!

I think we should make this solver symbolic for obstacles, goal and initial positions so that it can deal with randomized scenarios.

Additionally, i think many of the printouts can be removed. 
You could consider using `logging.debug` to not loose the value of some of these printouts.

File: plannerbenchmark/planner/acadosMpc/pointMassModel.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    
+    start = np.array(exp.initState()[0][:2])
+    goal = np.array(exp.goal().primeGoal().position())
+
+    robot_size = np.array([exp.rBody(), exp.rBody()])
+    obs_size = np.array([exp.obstacles()[0].radius(), exp.obstacles()[0].radius()])
+    print(robot_size)
+    print(obs_size)
+    w_pos = pr.w_pos
+    obs_pos = np.array(exp.obstacles()[0].position())
```
[the remark this answers is not in the document]
Comment (maxspahn) [reply plannerbenchmark/planner/acadosMpc/pointMassModel.py:41]: Actually only line 41 is important for that comment.

**Comment by c-salmi** [comment]
Still did not manage to make the start, goal and obs symbolic The way to do this in acados is not yet clear to me. But it does now work with random static obstacles and goals.

The problem was in the `__init__` function of the Planner class. There I used the `exp.Obstacles()` function of the Experiment class to get the info of the obstacles. However when random obstacles are requested, this function will not yet return the correct obstacles. Only when calling `exp.Obstacles()` at a later stage (like in `computeAction`) will it return the random obstacles. 

For now I moved the initialization of the planner to the first call to `computeAction`.

**Pull request merged**